shared/constants/urls.ts

```diff
@@ -1,3 +1,7 @@
 export enum BaseUrl {
   Portfolio = 'https://portfolio.metamask.io',
 }
+
+export enum SurveyUrl {
+  BtcSupport = 'https://www.getfeedback.com/r/yG6FbiW5',
+}
```

Comment:
Since it's flask only, do we want to have code fence here too? Otherwise this might be considered deadcode (which might be ok?)